### ISSUE TITLE
adaptors to "recruit"

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#17155]
I want to describe
"Next, Mid1 recruits other cytokinetic node proteins, including the myosin II Myo2, myosin light chain Rlc1, IQGAP Rng2, F-BAR Cdc15, and formin Cdc12, and Mid1 is incorporated into the CR along with type 2 node components [9, 11–17]."

(and many similar things, like recruitment for kinetochore assembly)

NTR protein recruitment activity
This would be defined
The binding activity of a molecule that localizes a protein (or ?) to a specific cellular component or region. This binding activity usually requires specific post-translational modifications and is required to assemble macromolecular structures in a coordinated way.

This would have a parentage BP  to cellular protein localization.

This way we could use

"protein recruitment activity" part of "cytokinetic ring assembly"
"protein recruitment activity" part of "kinetochore assembly"
 
and we wouldn't need to describe the localization (which is important but isn't  really the 'process' we are trying to capture.

Examples:
 Almonacid, M., Celton-Morizur, S., Jakubowski, J.L., Dingli, F., Loew, D., Mayeux, A., Chen, J.-S., Gould, K.L., Clifford, D.M., and Paoletti, A. (2011). Temporal control of contractile ring assembly by Plo1 regulation of myosin II recruitment by Mid1/anillin. Curr. Biol. 21, 473–479.

Paxillin-Mediated Recruitment of Calcineurin to the Contractile Ring Is Required for the Correct Progression of Cytokinesis in Fission Yeast.
Martín-García R, Arribas V, Coll PM, Pinar M, Viana RA, Rincón SA, Correa-Bordes J, Ribas JC, Pérez P.
Cell Rep. 2018 Oct 16;25(3):772-783.e4. doi: 10.1016/j.celrep.2018.09.062.
PMID: 30332655 

At present I'm using "protein binding" but this  is suboptimal, because a) it isn't directional, and b) it doesn't capture the localization aspect. We often make this with another different annotation, but this is really saying the same thing and we could remove this redundancy.

**Comment by ValWood** [comment]
This has come up in quite a few tickets, also  I tried to document this recently on the PomBAse tracker  for "chromatin recruitment" 
https://github.com/pombase/curation/issues/2117
Solution here was the same.

**Comment by ValWood** [comment]
We can see how inconsistent this is here:

![contractile](https://user-images.githubusercontent.com/7359272/56126773-95c36880-5f73-11e9-8b1a-d4be1a86d747.jpg)


most would become
"protein recruitment activity" part of "mitotic actimyosin cytokinetic ring assembly"

**Comment by ValWood** [comment]
It's important to be able to model all of the localization correctly because this is a very dynamic process. The components move around during different cell cycle phases.


"When constriction begins, Mid1 leaves the CR and re-accumulates in the nucleus, while type 2 node proteins remain in the CR [8]. Meanwhile, type 1 nodes reset to daughter cell middles to ensure medial cell division in the subsequent cell cycle [8, 18]. Although Mid1’s localization to type 1 and cytokinetic nodes depends on its ability to bind the plasma membrane and additional protein interactions modulated by Cdk1 and polo-like kinase Plo1 [7, 9, 10, 19–21], it was previously unknown how or why Mid1 leaves the division site at constriction onset.

Without clear terms for the MF it becomes impossible to model these processes

**Comment by pgaudet** [comment]
Can experiments distinguish between 'recruitement activity' and adaptor activity'? 
Or are you saying (based on the ticket title) that recruitment is a type of adaptor ? 

Thanks, Pascale

**Comment by ValWood** [comment]
I think this would be  a type of adaptor. but adaptors I'm not sure that adaptors are *always* recruiting.

If we think they *are* always recruiting this should be  added to the definition.
Maybe they are?

**Comment by ValWood** [comment]
GO:0030533    triplet codon-amino acid adaptor activity

is this "recruiting"

**Comment by ValWood** [comment]
GO:0005483    soluble NSF attachment protein activity
(I'm not sure if this is recruiting, it doesn't seem to align very well with "adaptor")

**Comment by pgaudet** [comment]
@thomaspd any comments on this one ? 
For transcription 'recruitment' is also an important step.

**Comment by pgaudet** [comment]
@ValWood you were going to look into this one further - any progress?

**Comment by ValWood** [comment]
I think we have already made good progress. I have MF terms  to use for all the examples I came across so far.

and we now have

chromatin adaptor (GO:0140463)
The binding activity of a molecule that brings together a protein or a protein complex with a nucleosome, to establish or maintain the chromatin localization of the protein, or protein complex. [PMID:32277274]

which mentions 'localization' and so is clearly 'recruitment'
Maybe the solution is to  add 

GO:0071168   protein localization to chromatin, as a BP link for this function term ?

In these situations I don't make the protein localization terms anymore as this information is captured by the activity (and  really it is often just how the signalling pathways work).

I think we can now close this and open tickets for specific refinements.
